docs/docs/keys/developers.md

```diff
@@ -36,13 +36,13 @@ A good choice for a remote signer is [Web3signer](https://docs.web3signer.consen
 
 ## Key Management Recommendation for Developers
 
-The Node can implement a feasible and sufficient method of loading the keys. This is asking for a path to a keystore folder. This keystore needs to follow some structure either the Service knows how to read, or an AVS [plugin](../plugin/intro.mdx) knows how to transform to something the Service knows how to read.
+The AVS can implement a feasible and sufficient method of loading the keys. This is asking for a path to a keystore folder. This keystore needs to follow some structure either the AVS knows how to read, or an AVS [plugin](/docs/category/plugin) knows how to transform to something the AVS knows how to read.
 
-Usually, the plugin idea is the most recommended because the Service has the freedom to use whatever structure they need or want for the keystore. Also, no constraint is being put on users or key generation tools. This also fits the idea of the [AVS setup wizard tool](../wizard/intro) supporting such plugins. Apart from the wizard domain, users could use this plugin (as a Service command/subcommand) to process the raw keystore if they go for a manual setup.
+Usually, the plugin idea is the most recommended because the AVS has the freedom to use whatever structure they need or want for the keystore. Also, no constraint is being put on users or key generation tools. This also fits the idea of the [AVS setup wizard tool](/docs/category/avs-setup-wizard) supporting such plugins. Apart from the wizard domain, users could use this plugin (as a AVS command/subcommand) to process the raw keystore if they go for a manual setup.
```

Comment:
```suggestion
Usually, the plugin idea is the most recommended because the AVS has the freedom to use whatever structure they need or want for the keystore. Also, no constraint is being put on users or key generation tools. This also fits the idea of the [AVS setup wizard tool](/docs/category/avs-setup-wizard) supporting such plugins. Apart from the wizard domain, users could use this plugin (as an AVS command/subcommand) to process the raw keystore if they go for a manual setup.
```

Reply:
Done [1571c70](https://github.com/NethermindEth/el-node-spec/pull/54/commits/1571c70ef22b23cea60709f36903c7b78dd3059b)